data/settings/src/main/java/com/google/jetpackcamera/settings/LocalSettingsRepository.kt

```diff
@@ -129,6 +130,37 @@ class LocalSettingsRepository @Inject constructor(
         }
     }
 
+    override suspend fun updateMaxFrameRate(
+        maxFrameRate: Int,
+        currentTargetFrameRate: TargetFrameRate
```

Comment:
I couldn't find where `TargetFrameRate` is defined -- is that part of this PR or already landed?

Reply:
Ah nevermind, I see it now